src/main/org/deidentifier/arx/distributed/Main.java

```diff
@@ -157,6 +157,7 @@ public boolean accept(File dir, String name) {
     public static void main(String[] args) throws IOException, RollbackRequiredException, InterruptedException, ExecutionException {
         System.out.println("Running multiple benchmarks overwrites results from previous runs!");
         System.out.println("You can pass following four required arguments: measureMemory?, testScalability?, datasetName, sensitiveAttribute");
+        ARXPartition.setRandomSeed(93981238859L);
```

Comment:
OK, Now I realize that this is a static method (see comment above).

I'm generally fine with this, but I think it should then be renamed, e.g. into "makeDeterministic(...)" or so.